app/controllers/application_controller.rb

```diff
@@ -3,6 +3,8 @@ class ApplicationController < ActionController::Base
 
   before_action :get_favorite#, only: [:new, :create]
 
+  add_flash_types :error
```

Comment:
would like to eventually play with the different notifcation ids like :notice or something